Example/Settings/FlagsmithSettings.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Example.Settings
+{
+    public class FlagsmithSettings
```

Comment:
I'm not too sure why we need a separate Settings class in the Example project. Shouldn't this just be part of the main project?

Reply:
`FlagsmithSettings ` is used to get and set flagsmith properties from app configuration only.

Reply:
Do we still need this? Certainly it shouldn't still reference `UserLegacyIdentities`, right?

Reply:
yes just used example to model properties from `appsettings.json` .

`var settings = configuration.GetSection("FlagsmithConfiguration").Get<FlagsmithSettings>();`